docs/buildtools/poetry/help.go

```diff
@@ -0,0 +1,12 @@
+package poetry
+
+var Usage = []string{"poetry <poetry args> [command options]"}
+
+func GetDescription() string {
+	return "Generate poetry build configuration."
```

Comment:
This command description seems to be wrong. It looks like this is the "poetry-config" description.